File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PackagesUpdateDecontaminateRequestItem.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PackagesUpdateDecontaminateRequestItem
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("DecontaminationDate")]
        public string? DecontaminationDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("DecontaminationMethodName")]
        public string? DecontaminationMethodName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("DecontaminationSteps")]
        public string? DecontaminationSteps { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PackageLabel")]
        public string? PackageLabel { get; set; }
    }
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdateDecontaminateRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesUpdateDecontaminateRequestItem(
    @JsonProperty("DecontaminationDate")
    val decontaminationDate: String? = null,
    @JsonProperty("DecontaminationMethodName")
    val decontaminationMethodName: String? = null,
    @JsonProperty("DecontaminationSteps")
    val decontaminationSteps: String? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null
)
